src/core/ext/xds/xds_client_stats.cc

```diff
@@ -131,29 +131,43 @@ XdsClusterLocalityStats::~XdsClusterLocalityStats() {
 
 XdsClusterLocalityStats::Snapshot
 XdsClusterLocalityStats::GetSnapshotAndReset() {
-  Snapshot snapshot = {
-      GetAndResetCounter(&total_successful_requests_),
-      // Don't reset total_requests_in_progress because it's
-      // not related to a single reporting interval.
-      total_requests_in_progress_.load(std::memory_order_relaxed),
-      GetAndResetCounter(&total_error_requests_),
-      GetAndResetCounter(&total_issued_requests_),
-      {}};
-  MutexLock lock(&backend_metrics_mu_);
-  snapshot.backend_metrics = std::move(backend_metrics_);
+  Snapshot snapshot;
```

Comment:
> No need for the = {}.
I don't see we initialize Snapshot fields, so believe total_* counters are undefined today. Instead I updated to explicitly initialize fields in the definition, I see no reason not to.